pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -121,11 +166,31 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 	}
 
 	machineState := s.GetMachineState()
+	reusable := p.memoryToReuse[string(pod.UID)]
+	// requested memory - reusable 
+	requestedAbsolute := make(map[v1.ResourceName]uint64)
+
+	for resourceName, resourceSize := range requestedResources {
+		//copy a map
+		requestedAbsolute[resourceName] = resourceSize
+
+		if _, ok := reusable[resourceName]; !ok {
+			continue
+		} 
+
+		reusableSize := reusable[resourceName].Size
+
+		if resourceSize > reusableSize {
+			requestedAbsolute[resourceName] = resourceSize - reusableSize
+			continue
+		}
+		requestedAbsolute[resourceName] = 0
```

Comment:
should we reduce `reusableSize` by the resourceSize?
something like that
```golang
p.memoryToReuse[string(pod.UID)][resourceName].Size -= resourceSize
```
or updating it under https://github.com/mgjeong/kubernetes/pull/30/files#diff-4c005e5ddcd048f3c5db48df4fdf1fa4ae7318d0d54f3c0964d074910946a2d1R251 should be enough

Reply:
`p.updateMemoryToReuse()` do the job